spec/features/user_can_see_basic_profile_info_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "when user logs in" do
+  before do
+    Capybara.app = Apicurious::Application
+    stub_oauth
+
+    token = ENV['github_user_token']
+    stubbed_response = {:login=>"stovermc", :id=>15877604, :avatar_url=>"https://avatars0.githubusercontent.com/u/15877604?v=3", :followers_url=>"https://api.github.com/users/stovermc/followers", :following_url=>"https://api.github.com/users/stovermc/following{/other_user}", :starred_url=>"https://api.github.com/users/stovermc/starred{/owner}{/repo}", :subscriptions_url=>"https://api.github.com/users/stovermc/subscriptions", :organizations_url=>"https://api.github.com/users/stovermc/orgs", :repos_url=>"https://api.github.com/users/stovermc/repos", :name=>"Mark Stover", :public_repos=>33, :public_gists=>5, :followers=>8, :following=>13 }
+
+    allow_any_instance_of(GithubService).to receive(:user_by).with(token).and_return(stubbed_response)
+    user = GithubUser.find_user(token)
```

Comment:
this setup seems to be a little overkill. I think I probably only need  line 5 and 6 for this test.  Line 8 through 12 I will need for stubbing a user when I create a method #get_profile_picture test.